crates/vault-registry/src/lib.rs

```diff
@@ -120,7 +120,7 @@ pub mod pallet {
 
         /// Currency used for griefing collateral, e.g. DOT.
         #[pallet::constant]
-        type GetGriefingCollateralCurrencyId: Get<CurrencyId<Self>>;
+        type GetGriefingCurrencyId: Get<CurrencyId<Self>>;
```

Comment:
I think the previous name was more explicit. Would keep it that way, but not super important.